modular_skyrat/modules/primitive_cooking_additions/code/cauldron.dm

```diff
@@ -0,0 +1,339 @@
+/obj/machinery/cauldron
+	name = "stone cauldron"
+	desc = "Cooks and boils stuff the old fashioned way."
+	icon = 'modular_skyrat/modules/primitive_cooking_additions/icons/stone_kitchen_machines.dmi'
```

Comment:
Don't put stuff in the modular_skyrat folder, put it in modular_zubbers instead, and make sure to put all icons in the base zubber's icon folder or iconforge will fail to load it